src/Pages/Overview/Overview.js

```diff
@@ -1,7 +1,146 @@
 import React, { Component } from 'react'
+import './Overview.css'
+import axios from 'axios'
+import OverviewProfile from './OverviewProfile.js'
+
+export default class OverviewBoard extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      // All users array, update by callDatabase
+      users: [],
+      query: '',
+      queryResult: []
+    }
+  }
+
+  componentDidMount () {
+    if (this.props.user) {
+      this.setState(
+        {
+          authToken: this.props.user.token,
+          currentUser: this.props.user.email,
+          currentUserLevel: this.props.user.accessLevel
+        },
+        () => {
+          this.callDatabase()
+        }
+      )
+    }
+  }
+
+  callDatabase () {
+    axios
+      // get all user!
+      .post('/api/user/users', {
+        // don't need email
+        token: this.state.authToken
+      })
+      .then(result => {
+        if (result.status >= 200 && result.status < 300) {
+          this.setState({ users: result.data })
+        }
+      })
+      .catch(err => {
+        console.log(err)
+      })
+  }
+
+  updateQuery (event) {
+    let { value } = event.target
+    value = value.trim().toLowerCase()
+    this.setState({ query: value })
+
+    if (!value) return
+    const queryResult = this.state.users.filter(data =>
+      data.firstName.toLowerCase().includes(value)
+    )
+    this.setState({ queryResult })
+  }
+
+  /*
+  Delete api
+  parameter: Json object of object to be deleted
+  */
+  deleteUser (user) {
+    // if they decide to delete themselves: delete->logout
+    axios
+      .post('/api/user/delete', {
+        token: this.state.authToken,
+        email: user.email
+      })
+      .then(result => {
+        this.callDatabase() // reload database
+      })
+      .catch(err => {
+        console.log(err)
+      })
+    if (user.email === this.state.currentUser) {
+      // logout
+      window.localStorage.removeItem('jwtToken')
+      window.location.reload()
+      return window.alert('Self-deprecation is an art')
+    }
+  }
 
-export default class extends Component {
   render () {
-    return <div>Overview page</div>
+    return (
+      <div className='layout'>
+        <h1>Users Dashboard</h1>
+
+        <h6>Search by first-name</h6>
+        <input
```

Comment:
Replace this input with a component I wrote a while back

[SearchBar](https://github.com/evanugarte/InventoryEcho/blob/master/client/src/components/SearchBar.js)

[SearchDropdown](https://github.com/evanugarte/InventoryEcho/blob/master/client/src/components/SearchDropdown.js)

See the relationship between the two components. they will take care of the UI for searching by first/last name, email, etc.

Reply:
moved this to the new issue I made. I will make another PR for it.